sql/item.h

```diff
@@ -4051,6 +4051,7 @@ class Item_insert_value : public Item_field
   bool eq(const Item *item, bool binary_cmp) const;
   bool fix_fields(THD *, Item **);
   virtual void print(String *str, enum_query_type query_type);
+  enum Type type() const { return INSERT_VALUE_ITEM; }
```

Comment:
For 5.7, wait until the 5.7.21 upstream merge and rebase, it introduced the same. For 5.6 it would apply as well but due to GCA it will have to be resolved at merge time